features/testbot/trip.feature

```diff
@@ -48,12 +65,85 @@ Feature: Basic trip planning
             | kl    |
             | la    |
 
+        When I plan a trip I should get
+            | waypoints               | trips         |
+            | a,b,c,d,e,f,g,h,i,j,k,l | alkjihgfedcba |
+
+    Scenario: Testbot - Trip: Roundtrip FS waypoints (more than 10)
+        Given the node map
+            """
+            a b c d
+            l     e
+            k     f
+            j i h g
+            """
+
+        And the ways
+            | nodes |
+            | ab    |
+            | bc    |
+            | de    |
+            | ef    |
+            | fg    |
+            | gh    |
+            | hi    |
+            | ij    |
+            | jk    |
+            | kl    |
+            | la    |
+
+        When I plan a trip I should get
+            | waypoints               | trips         | source |
```

Comment:
@TheMarex @danpat: and now... - fight! 😄
kidding. I actually agree with @danpat but in most of these cases we only test one parameter combination so the argument is just semi-valid.

If you both don't mind and there are no further opinions on these, I'd leave it as it is just because it's less work.